unit/solvers/lowering/byte_operators.cpp

```diff
@@ -158,6 +158,8 @@ SCENARIO("byte_extract_lowering", "[core][solvers][lowering][byte_extract]")
       // pointer_typet(u64, 64),
       vector_typet(u8, size),
       vector_typet(u64, size),
+      // seems to generate the correct value, but the simplifier needs to learn
+      // about complex_exprt, complex_real_exprt, complex_imag_exprt
       // complex_typet(s16),
       // complex_typet(u64)
```

Comment:
"Use #if 0 for commenting out source code"

Reply:
Done.